vnext/local-cli/generator-windows/templates/cpp/src/App.cpp

```diff
@@ -31,9 +32,9 @@ App::App() noexcept
     InstanceSettings().EnableDeveloperMenu(false);
 #endif
 
-    PackageProviders().Append(make<ReactPackageProvider>()); // Includes all modules in this project
+    REGISTER_AUTOLINKED_NATIVE_MODULE_PACKAGES(); // Includes any autolinked modules
 
-    REACT_REGISTER_NATIVE_MODULE_PACKAGES(); //code-gen macro from autolink
```

Comment:
>REACT_REGISTER_NATIVE_MODULE_PACKAGES [](start = 4, length = 37)

Existential question for all:
Should we make sure that the samples and test we have in the repo, today:
 * \packages\microsoft-reactnative-sampleapps
 * \packages\E2ETest

be kept up to date with any changes we make to the template. My humble opinion would be yes, but clearly this line actually never made it.

Reply:
This also ties into the upgrade story for customers I've tried to elude to.

---
In reply to: [421610548](https://github.com/microsoft/react-native-windows/pull/4816#discussion_r421610548) [](ancestors = 421610548)

Reply:
potentially yes, open an issue